app/controllers/app/base_controller.rb

```diff
@@ -0,0 +1,4 @@
+class App::BaseController < ApplicationController
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Not covered lines: 1 to 4